src/components/home/components/ContactForm.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Form, Icon, Input, Modal } from 'antd';
+import PropTypes from 'prop-types';
+
+import { INPUTS, CONTACT_STRINGS } from '../constants';
+
+const FormItem = Form.Item;
+const { TextArea } = Input;
+const { TITLE, PLACEHOLDER } = CONTACT_STRINGS;
+
+function generateContactInputs(decorator) {
+    return INPUTS.map(input => {
+        const { field, iconType, placeholder, rules } = input;
+
+        return (
+            <FormItem
+                key={field}
+            >
+                {
+                    decorator(field, {
+                        rules,
+                    })(
+                        <Input
+                            placeholder={placeholder}
+                            prefix={<Icon type={iconType} />}
+                        />
+                    )
+                }
+            </FormItem>
+            
+        );
+    });
+}
+
```

Comment:
I think it could look better and shorter if you destruct

![image](https://user-images.githubusercontent.com/34488930/51427651-b1c89f80-1bfa-11e9-844f-31ec12bdcab3.png)